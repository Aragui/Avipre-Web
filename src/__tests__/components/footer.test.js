import {render, screen} from '@testing-library/react'
import Footer from '../../components/footer'
import '@testing-library/jest-dom'

describe('<Footer />', () => {
    test('Render Footer', () => {
        const {container} = render(<Footer/>);
        expect(container).toMatchSnapshot();
    });
});
