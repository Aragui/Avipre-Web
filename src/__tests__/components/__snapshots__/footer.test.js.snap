// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Render Footer 1`] = `
<div>
  <div
    class="footer"
  >
    <div
      class="container"
    >
      <div
        class="row"
      >
        <div
          class="col-xl-3 col-md-6 col-sm-12 d-flex justify-content-start align-items-center"
        >
          <img
            alt="Avipre"
            class="ms-5"
            src="./images/logos/logotipo-2322x688-13.png"
            style="height: 70px;"
          />
        </div>
        <div
          class="col-md-4 col-sm-12 d-flex justify-content-start align-items-center"
        >
          <div
            class="ms-5 mt-3"
          >
            <h5>
              Dirección
            </h5>
            <p>
              C. 28 332C, entre 25 y 27
            </p>
            <p>
              Emiliano Zapata Nte, CP: 97129
            </p>
            <p>
              Mérida, Yuc., Mexico
            </p>
          </div>
        </div>
        <div
          class="col-xl-3 col-md-6 col-sm-12 d-flex justify-content-start align-items-center"
        >
          <div
            class="ms-5 mt-3"
          >
            <h5>
              Contacto
            </h5>
            <p>
              <strong>
                Arq. Johari Aviles Preciat
              </strong>
            </p>
            <p>
              Email: 
              <a
                class="mail"
                href="mailto:johari.aviles@gmail.com"
              >
                johari.aviles@gmail.com
              </a>
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
